module-2/lab-df-calculation-and-transformation/your-code/challenge-3.ipynb

```diff
@@ -32,7 +32,8 @@
    "source": [
```

Comment:
- How to select customers whose aggregated amount_spent is in a given quantile range?: You are asked to get the **list of customers** whose aggregated amount is in a certain percentile/quantile. At the beginning of the exercise 75%, 95% are requested.  
- When you do the following for labeling VIP and preferred customers, I think the output is not correct because of the conditional. I guess that the conditional for above 75 percentile is including also the 95 percentile. 
```Python
def label_customer(x, p75=percentile_75, p95=percentile_95): 
    res = 'Normal'
    if x['amount_spent'] > float(p75): 
        res = 'Preferred'
    if x['amount_spent'] > float(p95): 
        res = 'VIP'
    return res
    
customers['customer_label'] = customers.apply(lambda x: label_customer(x), axis=1)
customers.head(10)
```
-Q2 is not correct: You need to count distinct customers for each customer label and country. You are doing it counting orders instead of customers. 
-Q3: It's happening the same thing as in Q2 you are counting orders instead of distinct customers. 
Let's see it this afternoon in case after the revision with Marc!!